drivers/can/can_stm32fd.c

```diff
@@ -152,9 +152,8 @@ static int can_stm32fd_set_timing(const struct device *dev,
 	return can_mcan_set_timing(mcan_cfg, timing, timing_data);
 }
 
-static void can_stm32fd_line_0_isr(void *arg)
+void can_stm32fd_line_0_isr(const struct device *dev)
```

Comment:
Did you mean to drop `static`?

Reply:
good catch, no indeed. Will check, that's not supposed to happen obviously